build-support/scripts/oss-ent-shared-file-drift-detection.sh

```diff
@@ -0,0 +1,69 @@
+#!/usr/bin/env bash
```

Comment:
You can remove this file (I don't think it should go in OSS). I assume git included it for you 🙂